dataactbroker/handlers/fileHandler.py

```diff
@@ -959,14 +976,15 @@ def publish_fabs_submission(submission):
         response_dict = {'submission_id': submission_id}
         return JsonResponse.create(StatusCode.OK, response_dict)
 
-    def build_file_map(self, file_dict, file_type_list, upload_files, submission):
+    def build_file_map(self, file_dict, file_type_list, upload_files, submission, existing_submission=False):
```

Comment:
Instead of adding another parameter and making this less agnostic to the submissions and file types, could this logic be outside of the call and simply update the incoming `file_type_list` to exclude appropriations when it's not an existing submission? It's not a pressing point, just felt like `file_type_list` could do the same and keep the function simpler.